f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -239,6 +241,24 @@ def _pre_create(self, service):
             self.driver.network_builder.prep_service_networking(
                 service, traffic_group)
 
+    def _check_nonshared_network(self, service):
+        loadbalancer = service["loadbalancer"]
+        tenant_id = loadbalancer['tenant_id']
+
+        lb_net_id = loadbalancer['network_id']
+        network = self.driver.service_adapter.get_network_from_service(
+            service, lb_net_id)
+
+        if not self.driver.network_builder.l2_service.is_common_network(
+                network):
+            net_project_id = network["project_id"]
+            if tenant_id != net_project_id:
+                raise f5_ex.ProjectIDException(
+                    "\nThe tenant project id is %s."
```

Comment:
We should split log lines into 2, instead of add'\n'. It's not benefit for log search.